resources/assets/less/base.less

```diff
@@ -187,7 +187,7 @@ select.form-control {
     box-shadow: 0 10px darken(@secondary, 10%), 0 7px 10px fade(#000, 80%);
   }
 
-  &:active {
+  &:active, &.active {
```

Comment:
you mean the usage of buttons as tabs? i'm not really a designer 💃 , maybe flyte could make something nice.